src/resource_dasm.cc

```diff
@@ -2448,16 +2421,23 @@ Resource disassembly output options:\n\
   --save-raw=yes (or just --save-raw)\n\
       Save raw files even for resources that are successfully decoded.\n\
   --filename-format=FORMAT\n\
-      Specify the directory structure of the output. FORMAT can be one of the\n\
-      following values, which produce output filenames like these examples:\n\
+      Specify the directory structure of the output. FORMAT is a printf-like\n\
+      string with the following format specifications:\n\
+        %f:     the filename portion of 'input_filename'\n\

```

Comment:
Perhaps "the name of the file containing the resource" would be clearer here?

Reply:
Fixed.